libparsec/crates/protocol/tests/serialization.rs

```diff
@@ -0,0 +1,12 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+// TODO: Remove me
+#![allow(unused)]
```

Comment:
Why unused ? explain more clearly in TODO

Reply:
I think this was only temporary, I'll remove it.

Reply:
Actually, it is better to keep it to avoid warnings since all the test files contain unused code (at least for the moment). 
I added more details to the comment.